Makefile

```diff
@@ -0,0 +1,36 @@
+PROG_NAME := "esevaluator"
+
+setup: install tls
+
+clean:
+	go clean ./...
+
+install:
+	go get github.com/mitchellh/gox
+
+build:
+	go build -o $(GOPATH)/bin/$(PROG_NAME) ./cmd/esevaluator
+
+dist-build:
+	mkdir -p dist
+
+	gox -output="./dist/{{.OS}}-{{.Arch}}/$(PROG_NAME)" \
+		-os="linux windows darwin" \
+		-arch="amd64" \
+		./cmd/esevaluator > /dev/null
+
+docker: dist-build
```

Comment:
Stating this again in case the previous comment got lost. Remove `dist-build` from this target.